src/complexity_measures/statistical_complexity.jl

```diff
@@ -0,0 +1,192 @@
+using Distances
```

Comment:
Please reorganize this file in the following way: functions that are called by other functions should be defined **after the functions that call them**. This makes the code reading more intuitive and gives a better high level description of the code at the start rather than at the end.